pkgs/development/python-modules/py65/default.nix

```diff
@@ -20,21 +18,24 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace py65/tests/test_monitor.py \
-          --replace "test_argv_rom" "dont_test_argv_rom" \
-          --replace "test_argv_combination_rom_mpu" "dont_test_argv_combination_rom_mpu"
+        --replace-fail "test_argv_rom" "dont_test_argv_rom" \
+        --replace-fail "test_argv_combination_rom_mpu" "dont_test_argv_combination_rom_mpu"
   '';
 
+  build-system = [ setuptools ];
+
   meta = {
```

Comment:
Usually I put 
- homepage
- description
- longDescription
- everything else in ASCIIbetical